### DISCUSSION TITLE
Torrent not adding despite being a match

### DISCUSSION THREAD

**Discussion by arbitrarypunter** [cross-seed/cross-seed#194] [Q&A] [answered]
Hi

Been using cross-seed for a while - thanks for putting it together.

Got a couple of questions to try and get my head around why i'm seeing less cross seed's than expected.

Take this example, original torrent came from TorrentDay:

cross-seed search -v --torznab http://127.0.0.1:9696/prowlarr/4/api?apikey=d10c6e9338b444f1817acaf4ae8e5c8a  http://127.0.0.1:9696/prowlarr/1/api?apikey=d10c6e9338b444f1817acaf4ae8e5c8a http://127.0.0.1:9696/prowlarr/8/api?apikey=d10c6e9338b444f1817acaf4ae8e5c8a -i ~debian-transmission/test/input/ -s ~debian-transmission/test/output/
verbose: [configdump] {
  trackers: [],
  torznab: [
    'http://127.0.0.1:9696/prowlarr/4/api?apikey=[REDACTED]',
    'http://127.0.0.1:9696/prowlarr/1/api?apikey=[REDACTED]',
    'http://127.0.0.1:9696/prowlarr/8/api?apikey=[REDACTED]'
  ],
  torrentDir: '/var/lib/transmission-daemon/test/input/',
  outputDir: '/var/lib/transmission-daemon/test/output/',
  searchAll: true,
  includeEpisodes: false,
  fuzzySizeThreshold: 0.05,
  verbose: true,
  action: 'save',
  offset: 0,
  delay: 2
}
info: Validating your configuration...
info: Your configuration is valid!
info: Found 1 torrents, 1 suitable to search for matches
info: [1/1] Searching for Nine.Days.2160p.WEB-DL.DD5.1.HDR.HEVC-EVO
verbose: [torznab] http://127.0.0.1:9696/prowlarr/4/api?apikey=[REDACTED]&t=search&q=Nine+Days+2160p+WEB-DL+DD5+1+HDR+HEVC-EVO
verbose: [torznab] http://127.0.0.1:9696/prowlarr/1/api?apikey=[REDACTED]&t=search&q=Nine+Days+2160p+WEB-DL+DD5+1+HDR+HEVC-EVO
verbose: [torznab] http://127.0.0.1:9696/prowlarr/8/api?apikey=[REDACTED]&t=search&q=Nine+Days+2160p+WEB-DL+DD5+1+HDR+HEVC-EVO
verbose: [decide] Nine.Days.2160p.WEB-DL.DD5.1.HDR.HEVC-EVO.mkv - no match for IPTorrents torrent Nine Days 2021 2160p WEB-DL DD5 1 HDR HEVC-EVO - the info hash matches a torrent you already have (cached)
verbose: [decide] Nine.Days.2160p.WEB-DL.DD5.1.HDR.HEVC-EVO.mkv - no match for TorrentDay torrent Nine Days 2021 2160p WEB-DL DD5 1 HDR HEVC-EVO - the info hash matches a torrent you already have (cached)
verbose: [decide] Nine.Days.2160p.WEB-DL.DD5.1.HDR.HEVC-EVO.mkv - no match for TorrentLeech torrent Nine Days 2160p WEB-DL DD5 1 HDR HEVC-EVO - it has a different file tree (cached)
info: Done! Found 0 cross seeds from 1 original torrents

As you can see from the output, it found 3 potential matches for the torrent. 1 on TorrentDay - the original. 1 on IPTorrents, which i would have thought it would have matched for, but its saying the info hash matches so it isn't. I pulled this torrent down manually and compared it with cross-seed diff and it matches. And then the 3rd from TorrentLeech which has a different file tree (annoyingly just an extra nfo file!)

Why didn't it see the IPTorrents one as a legitimate match?

Also, what is the purpose of the fuzzySizeThreshold ? I had thought it might have been so that matches where things like missing .nfo files weren't considered a blocker to cross-seeding a torrent but clearly i have got that wrong! :-)

Any advice would be greatly appreciated.
Thanks

**Comment by zakkarry** [comment, marked as answer]
This is an old discussion, but providing an answer. 

You/clients don't typically allow torrents with the same infohash to be loaded into the same client. It causes false reporting of stats for both trackers combined to each tracker individually and is often seen as cheating to tracker staff.

This prevents that, and torrents on separate trackers are not supposed to share infohashes.
